### ISSUE TITLE
Draft a procedure for generating releases

### ISSUE THREAD

**Issue by goodmami** [globalwordnet/ili#4]
Issue #3 deals with the technical solution for creating mapping files, but there should be a document that explains when it is appropriate to do so. After a new wordnet is added to OMW? For a major OMW release? Other?

This document should also explain the steps required, including how to call the script, file naming guidelines, how (or whether) to [make a GitHub release](https://github.com/globalwordnet/ili/releases), post-generation steps such as announcements, etc.

**Comment by jmccrae** [comment]
Yes, I think that a release schedule would be a good idea. I think that a regular schedule (e.g., once/twice a year) would be ideal. 

For the first release can we at least try to include the Princeton WordNet 3.1 synsets?
